FromW3resource/src/basic3.py

```diff
@@ -1,6 +1,11 @@
 """
 3. Write a Python program to display the current date and time.
-Sample Output : 
-Current date and time : 
+Sample Output :
+Current date and time :
 2014-07-05 14:34:14
-"""
\ No newline at end of file
+"""
+
+import datetime
+#wrong format why? :print("Current date and time :\n",datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
+print("Current date and time :")
+print(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
```

Comment:
First "datetime" is the file's name of "datetime.py", second "datetime" is the class object of "datetime"